examples/advanced/inhibitory_gating.ipynb

```diff
@@ -8,10 +8,11 @@
   {
    "cells": [
     {
-     "cell_type": "markdown",
+     "cell_type": "heading",
+     "level": 1,
      "metadata": {},
      "source": [
-      "# Nengo Example: Inhibitory Gating of Ensembles"
+      "Nengo Example: Inhibitory Gating of Ensembles"
```

Comment:
I don't think I explicitly changed this. It might've auto-changed when I opened it in jupyter v4 and converted back to v3?